src/config.rs

```diff
@@ -134,26 +137,41 @@ impl std::fmt::Display for Config {
 #[cfg(test)]
 mod test {
     use super::*;
-    use std::path::Path;
+    use std::{path::Path, fs};
 
     #[test]
     fn simple() {
         let c: Config = toml::de::from_str(r#"
 user = "somebody"
-directory = "/path/to/somewhere"
+directory = "/etc/laurel/config.toml"
 statusreport-period = 86400
 [auditlog]
 file = "somefile"
 read-users = ["splunk"]
 "#).unwrap();
         println!("{:#?}", &c);
         assert_eq!(c.user, Some("somebody".to_string()));
-        assert_eq!(c.directory, Some(Path::new("/path/to/somewhere").to_path_buf()));
+        assert_eq!(c.directory, Some(Path::new("/etc/laurel/config.toml").to_path_buf()));
         assert_eq!(c.statusreport_period, Some(86400));
         assert_eq!(c.auditlog, Logfile{
             file: Path::new("somefile").to_path_buf(),
             users: Some(vec!["splunk".to_string()]),
             size: None, generations: None,
         });
     }
+// Run if you want to see wether or not your config file is parsed correctly
+    #[test]
+    fn simple_from_file() {
+        let config_file = fs::read_to_string("/etc/laurel/config.toml");
```

Comment:
This is the line that broke the CI test.

If you feel it's necessary to parse content from a file, please include it in the `testdata` subdirectory. Most likely, howver, you should be able to get away with parsing from an inline string (see `simple` test).